apple-pie/styles/style.css

```diff
@@ -0,0 +1,66 @@
+.box {
```

Comment:
To get rid of the white bars around the background image you can add this at the top of the file, before the `.box{` selector:

```
*{
    margin: 0
}
```